### PR TITLE
Sof 1835/bug aligned table sort arrows

### PR DESCRIPTION
**Pull request by vivanov90**
Aline Sofie table sort arrows to be in to the center of the table header row.

### CODE REVIEW

File: src/scss/_sofie-utility-classes.scss [hunk 1]
```diff
@@ -270,6 +270,14 @@
   visibility: hidden !important;
 }
 
+.relative {
+  position: relative !important;
+}
+
+.absolute {
+  position: absolute !important;
+}
```
Comment (LindvedKrvang) [inline src/scss/_sofie-utility-classes.scss:279]: Are you using this class?
We should try to refrain from adding code that we are not using. We can always add it once we actually need it.

File: src/app/shared/components/table/table.component.ts [hunk 2]
```diff
@@ -71,6 +71,10 @@ export class TableComponent<T> implements OnChanges {
     return header.sortDirection === SortDirection.ASC ? Icon.UPWARD_TRIANGLE : Icon.DOWNWARD_TRIANGLE
   }
 
+  public getSortIconTopPostion(header: SofieTableHeader<T>): number {
+    return header.sortDirection === SortDirection.ASC ? 3 : -3
```
Comment (LindvedKrvang) [inline src/app/shared/components/table/table.component.ts:75]: `3` and `-3` are magic numbers. Would be nice if they atleast got a variable with a name.

**Review by LindvedKrvang — COMMENTED** [review]
I know the icons are technically in the same position now, but given the nature of the two icons we are using, it still "look" like the icons are jumping up and down.

https://github.com/tv2/ng-sofie/assets/23049647/a8a4839d-d2ec-4d67-b989-07e0c852d546

The "base" of the icon (the horizontal line of the icon) should be aligned veritcallyl. This makes it look like that the icon just "flips" instead of looking like the icon is moving. 
Furthermore, the "base" should be placed so it is veritically centered with the label to the left.
To summarize: the "arrow-up" icon is centered correctly. The "arrow-down" icon needs to be moved further down.

File: src/scss/_sofie-utility-classes.scss (hunk 1 above)
Comment (vivanov90) [reply src/scss/_sofie-utility-classes.scss]: Removed

File: src/app/shared/components/table/table.component.ts (hunk 2 above)
Comment (vivanov90) [reply src/app/shared/components/table/table.component.ts]: Done

**Review by LindvedKrvang — APPROVED** [review]

**Review by RasmusAlbrektsen — APPROVED** [review]
